symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/pagination/PaginatedService.java

```diff
@@ -20,9 +21,20 @@
  */
 @API(status = API.Status.INTERNAL)
 public class PaginatedService<T> {
-  private PaginatedApi<T> paginatedApi;
-  private int chunkSize;
-  private int maxSize;
+  private final PaginatedApi<T> paginatedApi;
+  private final int chunkSize;
+  private final int maxSize;
+
+  public static StreamPaginationAttribute validatePaginationAttribute(StreamPaginationAttribute pagination, Integer defaultChunkSize, Integer defaultTotalSize) {
```

Comment:
Naming is quite confusing. Goal of this method is rather to create a StreamPaginationAttribute with default values if not set.

Reply:
do you have any naming suggestion for this method? 😅

Reply:
Something like `getValidPaginationAttribute` or `createValidPaginationAttribute`. I would assume a method beginning with `validate` to return void and to throw an exception if the passed object is not valid.